documentation/content/main/start-here/username-password.nuxt.md

```diff
@@ -316,16 +317,17 @@ Create `pages/index.vue`. This page will show the user's data. Redirect the user
 <!-- pages/index.vue -->
 
 <script lang="ts" setup>
-const { data } = await useFetch("/api/user");
-if (!data.value) throw createError("Failed to fetch data");
-const user = data.value.user;
-if (!user) await navigateTo("/login");
+const { data, error } = await useFetch("/api/user");
+if (error.value) throw createError("Failed to fetch data");
+const user = computed(() => data?.value?.user);
```

Comment:
we probably don't need to use a computed here as we are awaiting the fetch request above...